### ISSUE TITLE
Switchres has sorely inadequate documentation

### ISSUE THREAD

**Issue by chinagreenelvis** [libretro/docs#946]
## Description

Switchres doesn't work because I don't know how to make it work. All tutorials are outdated by years, and the documentation explains extraordinarily little about how to actually get it to function properly.

### Expected behavior

I expect to be able to find a resource that can explain to me very clearly how to get switchres to work on any television CRTs or VGA monitors I have connected to my computer.

### Actual behavior

I've spent an entire week trying to get it to work. On an RTX 3080, switchres does nothing. On a Radeon 5450 and CRTEmudriver, it causes Retroarch to crash.

### Steps to reproduce the bug

Literally just go here and read it and come away still knowing nothing: https://docs.libretro.com/guides/crtswitchres/

**Comment by gouchi** [comment]
@alphanu1 The documentation needs some update ? Thank you.

**Comment by chinagreenelvis** [comment]
So I've actually got it all figured out now, and I suppose this is stuff for advanced users anyway, but it did take me a couple of weeks to get it working. I would say that the documentation basically needs a straightforward tutorial, which at this point I could probably contribute to.

**Comment by hizzlekizzle** [comment]
That'd be awesome, actually! I think a lot of people would find that useful.

**Comment by sharmashivanand** [comment]
How did you get it to work?

**Comment by chinagreenelvis** [comment]
> How did you get it to work?

Core override files.

**Comment by sharmashivanand** [comment]
Can you please share or point me to some references? I have been trying to get thus working for weeks without any luck.

**Comment by chinagreenelvis** [comment]
Unfortunately there are so many software quirks to just about every step that writing a simple tutorial is impossible. The main documentation mentions how core override files are set up /cfg/corename/corename.switchres.ini

Here is the thread where I discovered the base INI file is being ignored:

http://forum.arcadecontrols.com/index.php/topic,168702.0.html

And here are my posts with the monitor presets I worked out for my SVGA monitor and my CRT TV:

https://forum.arcadecontrols.com/index.php/topic,116023.msg1772807.html#msg1772807

https://forum.arcadecontrols.com/index.php/topic,116023.msg1773029.html#msg1773029

If there's a particular point that you're getting hung up on you can send me a DM there and I can probably get you through it.

**Comment by zoltanvb** [comment]
It is indeed a bit tricky, as I also found out while testing a related change in RA.
Would it be better, if some info is added?
- an intro section (with similar content to https://github.com/antonioginer/switchres?tab=readme-ov-file#what-is-switchres )
- update that it works for Windows+AMD (with CRTEmuDriver, or it seems even without it), or X11 or KMS setups
- an explanation that this works with the normal video output which is usually HDMI in modern equipment, digital-to-analog conversion for actually connecting to a CRT is typically done by an external device
- an explanation that RetroArch will try to set up a mode where vertical resolution equals to the core resolution

**Comment by hizzlekizzle** [comment]
yeah, i think there's a lot of confusion about what it actually *does* and *doesn't* do, so that info would likely help.
